utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp

```diff
@@ -62,40 +62,150 @@ int main(int argc, char **argv)
     char funcType[1000] = {"Tensor_HOST_PKD3"};
 
     char funcName[1000];
+
+    //set user input
+
+    //set brightness default values
+    Rpp32f pBrightness[2] = {1.75, 50.0}; 
+
+    //set gamma correction default value
+    Rpp32f pGamma = 1.9;
+
+    //set blend default values
+    Rpp32f pBlend = 0.4;
+
+    //set exposure default values
+    Rpp32f pExposure = 1.4;
+
+    //set colorcast default values
+    uint pColorCastRGB[3] = {2, 3, 100};
+    Rpp32f pColorCastAlpha = 0.5;
+
+    //set colortwist default values
+    Rpp32f pColorTwist[4] = {1.4, 0.0, 60.0, 1.9};
+
+    //set crop default values
+    int pCrop[4] = {0, 0, 100, 180};
+
+    //Set CMN default values
+    int pCMNCrop[4] = {50, 50, 100, 100};
+    Rpp32f pCMNNormalize[2] = {0.0, 1.0};
+    Rpp8u pCMNMirror = 1;
+
+    //set colorjitter default values
+    Rpp32f pColorJitter[4] = {1.02, 1.1, 0.02, 1.3};
+
+    //set gridmask default values    
+    Rpp32u pGridMaskTileWidth = 40;
+    Rpp32f pGridMask[2] = {0.6, 0.5};
+    Rpp32u pGridMaskTranslate[2] = {0, 0};
+
+    //set spatter default values
+    uint pSpatterRGB[3] = {65, 50, 23};
+
     switch (test_case)
     {
     case 0:
         strcpy(funcName, "brightness");
+        if(argv[8] != NULL)
+        {
+            pBrightness[0] = atof(argv[8]);
```

Comment:
You may want to add small validator macros or functions (to validate if the user input is in the correct range) in a separate header file and include that here. A common header file that will have helper functions, #defines and includes common to all files.